assets/panel/debugger.properties

```diff
@@ -534,6 +534,13 @@ editor.addLogPoint.accesskey=l
 # LOCALIZATION NOTE (editor.editLogPoint): Editor gutter context menu item
 # for editing a log point already set on a line.
 editor.editLogPoint=Edit log
+editor.editLogPoint.accesskey=e
+
+
+# LOCALIZATION NOTE (editor.removeLogPoint): Context menu item for removing
+# a log point on a line.
+editor.removeLogPoint=Remove log
+editor.removeLogPoint.accesskey=p
```

Comment:
Why the external accesskey (not available in the label)?

Reply:
i don't follow?

Reply:
Label "Remove log", there's no "p", which will result in "Remove log (p)" (with p underscored) on Windows and Linux.